nsis/SharedCore.nsh

```diff
@@ -19,24 +19,10 @@ Function RedistInstallation
 ; Check if the VC runtimes are installed
 ReadRegDword $R5 HKLM "SOFTWARE\Microsoft\VisualStudio\14.0\VC\Runtimes\x86" "Installed"
 
-${If} $R5 == "1"
-    Goto DxSetup
-${EndIf}
```

Comment:
Replace with a `return` when `$R5` is present, it will skip the VC runtime from being downloaded/installed.